administrator/language/de-DE/plg_fields_note.ini

```
@@ -0,0 +1,17 @@
+; Joomla! German Translation
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2025 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_NOTE="Felder – Hinweis"
+PLG_FIELDS_NOTE_LABEL="Hinweis (%s)"
+PLG_FIELDS_NOTE_PARAMS_CLASS_DESC="Die Klasse, die für das div des Hinweises verwendet wird."
+PLG_FIELDS_NOTE_PARAMS_CLASS_LABEL="Hinweis-Klasse"
+PLG_FIELDS_NOTE_PARAMS_CLOSE_DESC="Dieses Feld steuert die Anzeige eines 'x' zum Schließen des Hinweises. Es nimmt den Wert 'true' (für Warnungen) oder den Wert für data-dismiss des Bootstrap-Schließen-Symbols an."
+PLG_FIELDS_NOTE_PARAMS_CLOSE_LABEL="Schaltfläche „Schließen“ anzeigen"
+PLG_FIELDS_NOTE_PARAMS_DESCRIPTION_LABEL="Hinweis-Inhalt"
+PLG_FIELDS_NOTE_PARAMS_DISPLAY_FRONTEND_LABEL="In Frontend anzeigen"
```

```suggestion
PLG_FIELDS_NOTE_PARAMS_DISPLAY_FRONTEND_LABEL="Im Frontend anzeigen"
```
https://github.com/joomlagerman/joomla/blob/8b3a255649b3624caa6d035996faa1428b9a01a4/administrator/language/de-DE/com_content.ini#L175-L176
https://github.com/joomlagerman/joomla/blob/8b3a255649b3624caa6d035996faa1428b9a01a4/language/de-DE/mod_menu.sys.ini#L10